x-pack/test/epm_api_integration/apis/template.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../api_integration/ftr_provider_context';
+import { getTemplate } from '../../../legacy/plugins/integrations_manager/server/lib/template/template';
+
+export default function({ getService }: FtrProviderContext) {
+  // This test was inspired by https://github.com/elastic/kibana/blob/master/x-pack/test/api_integration/apis/monitoring/common/mappings_exist.js
+  describe('load template', async () => {
+    const indexPattern = 'foo';
+    const templateName = 'bar';
+    const es = getService('es');
+
+    const body = getTemplate(indexPattern);
+
+    const response = await es.indices.putTemplate({
+      name: templateName,
+      body,
+    });
+    expect(response).to.eql({ acknowledged: true });
```

Comment:
I wonder if this assertion does anything that's not covered by the next `expect`. e.g. if the storage doesn't succeed, won't we find out when `getTemplate` doesn't return the expected results? 

Does our code require this response value? If ES were to respond with `{ success: true }` instead would our code fail / need to be updated?

If nothing else, I think it could use a comment since Sonja and I both had questions when first seeing it.

Reply:
If the first one is suceeding I hope the second suceeds too. But I rather have more checks then fewer which if things go wrong, I know where things went south.

For the return value: If it changes, we need to adjust our tests but this is also a breaking change in ES so I don't expect this to happen.

Happy to add the comment if is confusing. Already merged it as I don't think it is a blocker for the PR.